packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -259,10 +250,8 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       filled: {
         ...getFilledColorProps(
           {
-            backgroundColor: globalTokens.color.grey[92],
-            color: globalTokens.color.grey[38],
-            backgroundColorDark: t.colors.black,
-            colorDark: globalTokens.color.grey[68],
+            backgroundColor: t.colors.neutralBackground5,
+            color: t.colors.neutralForeground3,
             hcBackground: t.colors.neutralBackground3,
             hcColor: t.colors.brandForeground1,
             hcBorderColor: t.colors.brandForeground1,
```

Comment:
border for this badge is specified as transparent stroke (previously called stroke accessible). This would make it so that you don't need to specify any HC colors for this badge.